crates/ruma-client-api/CHANGELOG.md

```diff
@@ -45,6 +45,8 @@ Improvements:
 - Add `dir` `Request` field on the `get_relating_events_with_rel_types` and
   `get_relating_events_with_rel_type_and_event_type` endpoints
 - Add unstable support for moderator server support discovery, according to MSC4121
+- Add unstable support for MSC3266 behind the `unstable-msc3266` feature, and a new
+  `room::get_summary` module.
```

Comment:
This changelog entry makes it sound like the unstable MSC3266 and `room::get_summary` are separate.
```suggestion
- Add unstable support for the room summary endpoint from MSC3266 behind the
  `unstable-msc3266` feature.
```